cli/src/api_client/mod.rs

```diff
@@ -159,10 +160,14 @@ impl ApiClient {
                     },
                 )?;
 
-                response
-                    .json::<api::QuarantineConfig>()
-                    .await
-                    .context("Failed to get response body as json.")
+                let response_body = response.text().await?;
+                match serde_json::from_str::<api::QuarantineConfig>(&response_body) {
+                    Ok(config) => Ok(config),
+                    Err(e) => {
+                        log::error!("Error retrieving quarantine configuration");
+                        return Err(anyhow::anyhow!("Failed to parse response body as json: {}", e));
+                    }
+                }
```

Comment:
No need for this change. This is effectively the same thing, but much more verbose

Reply:
ah sorry forgot to clean this up, I broke it down for debugging to log the body